db/primary_migrate/20221012172457_alter_registration_logs_nullable_submitted_at.rb

```diff
@@ -0,0 +1,13 @@
+class AlterRegistrationLogsNullableSubmittedAt < ActiveRecord::Migration[7.0]
+  def up
+    safety_assured do
+      execute 'ALTER TABLE "registration_logs" ALTER COLUMN submitted_at DROP NOT NULL'
```

Comment:
Why no do with with ActiveRecord instead of raw SQL? I think `change_column_null` can do this

Reply:
because I just assumed this was the only way 😂 great catch, updated in 1f0352325ab73b3a7210c3283f61ae106372cfcd, PTAL!